pr_6/javascript/random_samples/direct-response-write.js

```
@@ -0,0 +1,184 @@
+    var footer = "</html>";
+    var output = header + msg + footer;
+    // ruleid: direct-response-write
+    res.send(output);
+});
+// {/fact}
+
+
+
+
+// {fact rule=cross-site-scripting@v1.0 defects=1}
+var express = require('express');
```

Biome (2.1.2)

[error] Shouldn't redeclare 'express'. Consider to delete it or rename it.

'express' is defined here:


(lint/suspicious/noRedeclare)